WebUI/Pages/Index.razor

```diff
@@ -1,4 +1,115 @@
 ﻿@page "/"
 @layout UnauthorizedPagesLayout
+@using ConestogaInsidersClub.Data.Models;
+@using ConestogaInsidersClub.Data.DataAccess;
+@using ConestogaInsidersClub.Pages.ViewModels;
 
-<h1>Games List</h1>
+@inject IGameService gameService;
+
+
+<h1>Conestoga Insiders Club</h1>
+<p>Your source of the latest games and events</p>
+
+<hr />
+
+@*--- Display Game List ---*@
+@if (showDetails)
+{
+    <br />
+    <h3>Game Details</h3>
+    <br />
+    <table>
+        <tr>
+            <td>Name: </td>
+            <td class="align-content-end"><span id="Game_Name">@game.Name</span></td>
+        </tr>
+        <tr>
+            <td>Description: </td>
+            <td><span id="Game_Name">@game.Description</span></td>
+        </tr>
+        <tr>
+            <td>Price: </td>
+            <td><span id="Game_Name">@game.Price.ToString("C")</span></td>
+        </tr>
+        <tr>
+            <td>Genre: </td>
+            <td><span id="Game_Name">@game.Genre</span></td>
+        </tr>
+    </table>
+    <br />
+    <AuthorizeView>
+        <Authorized>
+            @*TODO - Hook up game purchase and wishlist features*@
+            <button class="btn btn-primary">Buy Game</button>
+            <button class="btn btn-primary">Add To Wishlist</button>
+            <button class="btn btn-primary">Leave Rating</button>
+        </Authorized>
+    </AuthorizeView>
+    <button class="btn btn-secondary" @onclick="ToggleDetails">Go Back</button>
+
+}
+else if (games != null && games.Count != 0)
+{
+    <br />
+    <h3>Games</h3>
+    <p>Below you can find a selection of our available games.</p>
+    <br />
+    <div class="row" style="padding-top: 10px">
```

Comment:
You can also use `p-_number_` as a class (if you need some spacing and don't care about the specific numbers), since we have Bootstrap in this project and it provides different styles that we're using (such as `btn-primary`), including padding. But that's up to you, just makes the code look a bit cleaner.